README.md

```diff
@@ -77,6 +77,20 @@ load(
 protobuf_deps()
 ```
 
+### Note about rules_swift
+
```

Comment:
@tinder-maxwellelliott Happy Friday and thank you kindly for the PR!

I believe if you'd like to use a version of `rules_swift` different than what `rules_ios` pulls by default you can achieve that today w/o patching - just define `rules_swift` prior to defining `rules_ios`  in your `WORKSPACE`. Then, the repository definition of [`rules_swift` in `rules_ios`](https://github.com/bazel-ios/rules_ios/blob/master/rules/repositories.bzl#L60) will conditionally set that dependency. For instance:
```py
# WORKSPACE
git_repository(name="build_bazel_rules_swift",..)
git_repository(name="build_bazel_rules_ios",..)
```

Please LMK if you've got any questions!

Reply:
I will give this a try :facepalm:, closing for now